### ISSUE TITLE
Update current patch

### ISSUE THREAD

**Issue by foxy-munich** [AaronYang2333/LOL_Overlay_Assistant_Tool#3]
First of all, this is an excellent work. Is it possible to update the software to the latest patch. Or is der are manual to this by my one?

Best regards.

**Comment by AaronYang2333** [comment]
of course, you can, if u wanna update the tool, you might need to recapture the champion's profile and retain the model (all code are available on the build_dataset dir [https://github.com/AaronYang2333/LOL_Overlay_Assistant_Tool/tree/master/build_dataset])

**Comment by foxy-munich** [comment]
Ok thank you,.
I have two questions.  For example in the minimap_recognition the training image path and the valid image path, which images are inside this two folders?

And the second question, this folder (https://github.com/AaronYang2333/LOL_Overlay_Assistant_Tool/tree/master/poro_app/resources/data and https://github.com/AaronYang2333/LOL_Overlay_Assistant_Tool/tree/master/poro_app/resources/data/items ) contains a few csv files, do these have to be updated manually? And where do you get the Spearman correlation  r?

**Comment by AaronYang2333** [comment]
https://drive.google.com/file/d/1ZRdSPRJ5yB5BBd71Rf3_Sx-e3TPkfjT2/view?usp=sharing  training data

most data i crawled from https://euw.op.gg/

**Comment by AaronYang2333** [comment]
and you can use other measurements to compute the similarity, it doesn't matter.

**Comment by foxy-munich** [comment]
Thats perfect thank you. Is it possible to upload the other training dataset e.g.  in game profile, ban face?
Thank you for your efforts.

**Comment by foxy-munich** [comment]
> and you can use other measurements to compute the similarity, it doesn't matter.

Which values do you compare for the champbasestats correlation and for the wirkte?

**Comment by isetthestandard** [comment]
> and you can use other measurements to compute the similarity, it doesn't matter.

Can you tell me which data you have taken from op.gg for the winrate calculation?
